tricycle_steering_controller:

  wheel_separation:
    {
      type: double,
      default_value: 0.0,
      description: "Wheel separation.",
      read_only: false,
    }

  wheelbase:
    {
      type: double,
      default_value: 0.0,
      description: "Wheelbase length.",
      read_only: false,
    }

  wheel_radius:
    {
      type: double,
      default_value: 0.0,
      description: "Wheel radius.",
      read_only: false,
    }
